### PR TITLE
feat(EMI-1747): view artwork from notification panel for partner offers

### PR DESCRIPTION
**Pull request by leamotta**
The type of this PR is: **Feat**

This PR solves [EMI-1747]

### Description

Add new CTA to the alert panel to redirect collector to the artwork page, behind a feature flag.
Adjust designs according to new [Figma file](https://www.figma.com/file/RHkhhJJwixhFPltqhH9ZkW/v1.5-Saves-Partner-Offers?type=design&node-id=4381-8251&mode=design&t=bYxXnEPpiYPRurkx-0#757831092)

Screenshot 2024-04-04 at 11 48 02
Screenshot 2024-04-04 at 11 48 30
Screenshot 2024-04-04 at 11 49 09
Screenshot 2024-04-04 at 11 49 26
Screenshot 2024-04-04 at 11 52 33
Screenshot 2024-04-04 at 11 53 06

[EMI-1747]: https://artsyproduct.atlassian.net/browse/EMI-1747?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Components/Notifications/PartnerOfferArtwork.tsx [hunk 1]
```diff
@@ -130,15 +148,35 @@ export const PartnerOfferArtwork: FC<PartnerOfferArtworkProps> = ({
...
+        {partnerOfferVisibilityEnabled && fullyAvailable && (
+          <Button
+            // @ts-ignore
+            as={RouterLink}
+            to={artworkListingHref}
+            data-testid="partner-offer-view-artwork-button"
+            flex={1}
+            variant={"secondaryBlack"}
+          >
+            {"View Work"}
```
Comment (narikazu) [inline src/Components/Notifications/PartnerOfferArtwork.tsx:177]: According to [the design](https://www.figma.com/file/RHkhhJJwixhFPltqhH9ZkW/v1.5-Saves-Partner-Offers?type=design&node-id=2261-24754&mode=design&t=lGsYwiNY83j47vGL-4), this should be `View Artwork`. 

```suggestion
            {"View Artwork"}
```

**Review by narikazu — APPROVED** [review]
LGTM

File: src/Components/Notifications/PartnerOfferArtwork.tsx (hunk 1 above)
Comment (leamotta) [reply src/Components/Notifications/PartnerOfferArtwork.tsx:177]: the new designs are [here](https://www.figma.com/file/RHkhhJJwixhFPltqhH9ZkW/v1.5-Saves-Partner-Offers?type=design&mode=design&t=r6goUJCMzvewwlDV-0#757831092)

Comment (narikazu) [reply src/Components/Notifications/PartnerOfferArtwork.tsx:177]: Good to know! I thought the design for the note only got updated.

File: src/Components/Notifications/PartnerOfferArtwork.tsx [hunk 2]
```diff
@@ -130,15 +148,35 @@ export const PartnerOfferArtwork: FC<PartnerOfferArtworkProps> = ({
...
+        justifyContent="space-between"
+        width="100%"
+        maxWidth={CARD_MAX_WIDTH}
+      >
         <Button
           // @ts-ignore
           as={RouterLink}
           to={href}
           data-testid="partner-offer-artwork-button"
+          mr={partnerOfferVisibilityEnabled && fullyAvailable ? 2 : 0}
```
Comment (nickskalkin) [inline src/Components/Notifications/PartnerOfferArtwork.tsx:163]: I think you can use the `gap` property on the box above for margins

Comment (leamotta) [reply src/Components/Notifications/PartnerOfferArtwork.tsx:163]: Cool, I'll add it to the follow up changes, thanks!

**Pull request merged**